tests/jballerina-unit-test/src/test/resources/test-src/types/uniontypes/union_types_basic.bal

```diff
@@ -417,6 +417,25 @@ function testParenthesisedSingletonUnionType() {
     assertEquality(true, b);
 }
 
+function testValidUsageOfListBPInUnion() {
+    [string, boolean]|[int] [name1, married1] = fn1();
+    assertEquality(true, name1 is string);
+    assertEquality(true, married1 is boolean);
```

Comment:
These are redundant because the value equality checks below include checks for type.